roles/mongodb/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: install mongodb packages
+  apt: pkg={{ item }} state=installed
+  with_items:
+    - mongodb-org-server=3.0.6
+    - mongodb-org-shell=3.0.6
```

Comment:
Perhaps make the versions of these and pymongo a variable in defaults. Saves hunting for where to change them when version bumping is required.